docs/shared/features/powerpack/conformance.md

```diff
@@ -1 +1,58 @@
 # Run Language-Agnostic Conformance Rules
+
+The `@nx/powerpack-conformance` plugin allows [Nx Powerpack]() users to write and apply rules for your entire workspace that help with **consistency**, **maintainability**, **reliability** and **security**.
+
+The conformance plugin allows you to **encode your own organization's standards** so that they can be enforced automatically. Unlike ESLint, the rules are **applied to the whole workspace**, instead of one file at a time. The rules are written in TypeScript but can be **applied to any language in the codebase** or focus entirely on configuration files.
+
+The plugin also provides the following pre-written rules:
+
+- **Enforce Module Boundaries**: Similar to the Nx [ESLint Enforce Module Boundaries rule](/features/enforce-module-boundaries), but enforces the boundaries on every project dependency, not just those created from typescript imports or `package.json` dependencies.
+- **Ensure Owners**: Require every project to have an owner defined for the [`@nx/powerpack-owners` plugin](/nx-api/powerpack-owners)
+
+## Conformance Plugin Requires Nx Powerpack
+
+The `@nx/powerpack-conformance` plugin requires an Nx Powerpack license to function. [Activating Powerpack](/recipes/installation/activate-powerpack) is a simple process.
+
+{% call-to-action title="Buy a Powerpack License" icon="nx" description="Unlock all the features of Nx" url="https://nx.app/nx-powerpack/purchase" /%}
+
+## Configure Conformance Rules
+
+Conformance rules are configured in the `conformance` property of the `nx.json` file. You can use the pre-defined rules or reference [your own custom rule](/nx-api/powerpack-conformance#custom-conformance-rules).
+
+```jsonc {% fileName="nx.json" %}
+{
+  "conformance": {
+    "rules": [
+      {
+        "rule": "@nx/powerpack-conformance/enforce-module-boundaries",
+        "options": {
+          "depConstraints": [
+            {
+              "sourceTag": "scope:shared",
+              "onlyDependOnLibsWithTags": ["scope:shared"]
+            }
+          ]
+        }
+      },
+      {
+        "rule": "@nx/powerpack-conformance/ensure-owners",
+        "projects": ["!experimental-app"]
```

Comment:
Please add a small section on projects allowing you to opt projects in or out of conformance rules and the fact that all project matchers folks are used to from run-many are supported

Reply:
Oh I guess it's explained in the next .md doc, I guess maybe I'd change the request to be to reference that other .md file from this one, it doesn't seem to right now, I think we'd want to link to if for folks to learn more?